CHANGES.rst

```diff
@@ -4,4 +4,19 @@ Changelog
 1.0a1 (unreleased)
 ------------------
 
+- Fixed errors for code analysis test of this package
+  [macagua]
+
+- Pinned setuptools in buildout configuration for use latest version
+  [macagua]
+
+- Updated information about this package.
```

Comment:
README information like this https://github.com/Covantec/collective.opendata/commit/637f1dc95acb7f5819ca3e525c2751eff62f87a5